src/bio/bio_buffer.c

```diff
@@ -989,7 +1008,7 @@ bio_iod_post(struct bio_desc *biod)
 		dma_rw(biod, false);
 	else
 		biod->bd_result = 0;
-
+	D_DEBUG(DB_IO,"biod->bd_result %d", (int)biod->bd_result);
```

Comment:
(style)  space required after that ',' (ctx:VxV)

Reply:
removed the debug message.